tox.ini

```diff
@@ -1,5 +1,5 @@
 [tox]
-envlist = {py35,py38}-django{111,20,21,22}
```

Comment:
Did this come from cookiecutter?

Reply:
Yea, made https://github.com/edx/edx-cookiecutters/pull/65 to fix the cookiecutter as well.